fix(tem): blocklist (#4288)
scaleway/docs-content: faq/transactional-email.mdx

## How can I configure DNS records to send emails?

> You need to [add SPF and DKIM records](/managed-services/transactional-email/how-to/add-spf-dkim-records-to-your-domain/) to be able to send emails. Find out [how to set up your DNS records for other DNS providers](/tutorials/set-up-spf-dkim-for-dns-providers/).

Reviewer: ```suggestion
You need to [add SPF and DKIM records](/transactional-email/how-to/add-spf-dkim-records-to-your-domain/) to be able to send emails. Find out [how to set up your DNS records for other DNS providers](/tutorials/set-up-spf-dkim-for-dns-providers/).
```

## How do I know if my domain is valid for sending emails?

After configuring your SPF and DKIM records, you must launch a verification of the domain either via [the console](https://console.scaleway.com/transactional-email/domains) in the **DNS verification** tab, or the [API](https://www.scaleway.com/en/developers/api/transactional-email/#path-domains-display-spf-and-dkim-records-status-and-potential-errors). You will then receive an email informing you whether your domain was validated or if there are errors to fix so it can be validated.

## Do I have to set up an MX record for better email deliverability?

> Setting up an [MX record](/managed-services/transactional-email/concepts#mx-record) is highly recommended to ensure your email deliverability. Find out [how to add an MX record to your domain](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/#how-to-add-your-own-mx-record). You can also [use Scaleway's blackhole MX](/managed-services/transactional-email/how-to/add-mx-record-to-your-domain/#how-to-add-scaleways-mx-record) if you do not have your own MX.

Reviewer: ```suggestion
Setting up an [MX record](/transactional-email/concepts#mx-record) is highly recommended to ensure your email deliverability. Find out [how to add an MX record to your domain](/transactional-email/how-to/add-mx-record-to-your-domain/#how-to-add-your-own-mx-record). You can also [use Scaleway's blackhole MX](/transactional-email/how-to/add-mx-record-to-your-domain/#how-to-add-scaleways-mx-record) if you do not have your own MX.
```

## What happens to automatically configured DNS records when I delete a domain in the Transactional Email service?

> When a domain is removed from Scalway's Transactional Email platform, [autoconfigured DNS records](/managed-services/transactional-email/how-to/autoconfigure-domain/) are not automatically deleted in the Domains & DNS section. If you no longer use the domain for sending emails, you must manually delete or modify these records. It is recommended to verify the associated DNS records to ensure they are correctly configured or removed.

Reviewer: ```suggestion
When a domain is removed from Scalway's Transactional Email platform, [autoconfigured DNS records](/transactional-email/how-to/autoconfigure-domain/) are not automatically deleted in the Domains & DNS section. You must manually delete or modify these records if you no longer use the domain to send emails. Verifying the associated DNS records is recommended to ensure they are correctly configured or removed.
```

## What is the impact on the Transactional Email service if a DNS zone is deleted in Scaleway Domains & DNS?

> If a DNS zone is deleted while [autoconfig is enabled](/managed-services/transactional-email/how-to/autoconfigure-domain/), the DNS records will be added to the parent zone. This ensures that the domain remains validated on the Transactional Email platform, allowing continued use of the service without interruption.

Reviewer: ```suggestion
If a DNS zone is deleted while [autoconfig is enabled](/transactional-email/how-to/autoconfigure-domain/), the DNS records will be added to the parent zone. This ensures that the domain remains validated on the Transactional Email platform, allowing continued service use without interruption.
```

## What are the limitations and restrictions when using Transactional Email?

> Find out about [limits and quotas](/managed-services/transactional-email/reference-content/tem-capabilities-and-limits/) available for Transactional Email. If you want to increase your quota beyond the values shown on this page, [contact our support team](https://console.scaleway.com/support/create).

Reviewer: ```suggestion
Find out about [limits and quotas](/transactional-email/reference-content/tem-capabilities-and-limits/) available for Transactional Email. If you want to increase your quota beyond the values shown on this page, [contact our support team](https://console.scaleway.com/support/create).
```
